galaxy_ng/app/migrations/0019_pulp_container_access_policies.py

```diff
@@ -0,0 +1,193 @@
+from django.db import migrations
+import django_lifecycle.mixins
+import galaxy_ng.app.access_control.mixins
+
+
+viewsets = {
+    # Note. This is the default Pulp Continer access policy with some modifications.
+    # Our changes have been marked with comments.
+    "distributions/container/container": {
```

Comment:
The policy for container-push repos is omitted here (tag/untag/remove image) - this will be added later whenever image management will be enabled, correct?

Reply:
@ipanova are there things in the container push repo access policies that we should change? it looked like the defaults would work to me.

Reply:
clarified on the irc, resolved.